R/palaeorotate.R

```diff
@@ -92,6 +96,11 @@
 #'   - 0--540 Ma (grid)
 #'   - 0--750 Ma (point)
 #'
+#' While access is provided to all Global Plate Models available via the
+#' GPlates API, the use of the following mantle reference frame models is not
+#' recommended for reconstructing coordinates:
+#' "MULLER2022", "MULLER2019", and "MATTHEWS2016_mantle_ref".
```

Comment:
Given this, would it make sense to put the other ones first in the above documentation? I realize it's currently chronological, but it might make more sense to try to bring the recommended models to the forefront?

Reply:
This section has been updated. Let me know if you still have concerns!